### ISSUE TITLE
Package Control: Advanced Install Package 

### ISSUE THREAD

**Issue by blagoeres** [wbond/package_control#1191]
When running `Package Control: Advanced Install Package` I get an error in popup with the following message. 

> It appears a package is trying to ignore itself, causing a loop. Please resolve by removing the offending ignored_packages setting.

The popup appears few times and after I click OK it continues with installation. I use **3126** build.

![](https://i.imgur.com/mrks21c.gif)

**Comment by FichteFoll** [comment]
What is the text you insert in that box?

**Comment by wbond** [comment]
You need to provide a debug log (as directed when you created this issue).

**Comment by blagoeres** [comment]
@FichteFoll I inserted plugins that I want to install like - Schemr,Themr,SCSS,Seti_UI... as a comma separated list.

@wbond debug log is massive, so I avoided pasting it here at GitHub.
http://pastebin.com/j6rUQ0SK


The installation of packages goes fine after these pesky pop-ups.

**Comment by math2001** [comment]
Have you tried installing one plugin at a time? Check that you don't have a plugin twice in the list...

**Comment by blagoeres** [comment]
@math2001 Yes I am. With one plugin at a time there is no any problem. I use this way because it is easier when you have a lot of plugins.

**Comment by math2001** [comment]
> Yes I am

you are what? Sure that there is not a plugin twice in the list?

Have you tried with two plugin at the time? I know it sound stupid after my first message, but it's the two things to try: *one*, *two* and then we'll be able to say if it's your list that has a problem (maybe a plugin in the list) or package control that does not support multiple.

**Comment by blagoeres** [comment]
> you are what? Sure that there is not a plugin twice in the list?

Yes.

I tried to install 3 plugins now - `SublimeLinter,Color Highlighter,ColorPicker` and there was no problem.

Here is the full list of plugins when I get errors:

`SublimeLinter,Color Highlighter,ColorPicker,SideBarEnhancements,AutoFileName,Autoprefixer,Seti_UI,Theme - Spacegray,Dracula Color Scheme,Material Theme,Pug,Wild Cherry,Colorsublime,Haml,Handlebars,PackageResourceViewer,Emoji,ColorHelper,SublimeLinter-php,MarkdownEditing,Theme - Cobalt2,Emmet,Origami,ApplySyntax,Sass,SCSS,CSS3,Seti_UX,Predawn,Theme - Soda,AdvancedNewFile,Monokai Extended,EditorConfig,JavaScriptNext - ES6 Syntax,Themr,Schemr,SFTP,SublimeLinter-contrib-eslint`

**Comment by FichteFoll** [comment]
Can reproduced on a fresh portable ST3 version. The popup opened 12 times at the beginning and 2 times later during installation.

**Comment by jorgeas80** [comment]
Same problem here. How could I get a debug log?

**Comment by FichteFoll** [comment]
https://packagecontrol.io/docs/troubleshooting#Enabling_the_Debug_Log

**Comment by deathaxe** [comment]
Caused by various race conditions. Will be resolved by Package Control 4.
